src/prometheus-net.DotNetRuntime/EventListening/Parsers/KestrelEventParser.cs

```diff
@@ -0,0 +1,220 @@
+﻿using System;
+using System.Diagnostics.Tracing;
+using Prometheus.DotNetRuntime.EventListening.EventSources;
+using Prometheus.DotNetRuntime.EventListening.Parsers.Util;
+
+namespace Prometheus.DotNetRuntime.EventListening.Parsers
+{
+    public class KestrelEventParser : IEventParser<KestrelEventParser>, KestrelEventParser.Events.Info
```

Comment:
Do you know what version of .NET core this was introduced in?

Reply:
Ah, 3.1. Also meant to ask what verison of _asp.net core_ this was introduced in.